osquery/logger/plugins/windows_event_log_manifest/osquery.h

```diff
@@ -0,0 +1,66 @@
+//**********************************************************************`
+//* This is an include file generated by Message Compiler.             *`
+//*                                                                    *`
+//* Copyright (c) Microsoft Corporation. All Rights Reserved.          *`
+//**********************************************************************`
+#pragma once
+//+
+// Provider Facebook Event Count 5
+//+
+EXTERN_C __declspec(selectany) const GUID OsqueryWindowsEventLogProvider = {
+    0xf7740e18,
+    0x3259,
+    0x434f,
+    {0x97, 0x59, 0x97, 0x63, 0x19, 0x96, 0x89, 0x00}};
+
+//
+// Channel
+//
+#define OsqueryWindowsEventLogChannel 0x10
+
+//
+// Opcodes
+//
+#define _opcode_message 0xa
+
+//
+// Tasks
+//
+#define WindowsEventLogMessage 0x1
+//
```

Comment:
nit: throw a newline above this comment for consistency

Reply:
This file is automatically generated by the message compiler; should we still change it?